README.md

```diff
@@ -603,6 +655,23 @@ network_connections:
       rule_append_only: yes
 ```
 
+Configuring 802.1x:
+
+```yaml
+network_connections:
+  - name: eth0
+    type: ethernet
+    802.1x:
```

Comment:
I propose to change the name to `802_1x` or some other valid identifier, so that the name can be used in e.g. Jinja2. Dot has a special meaning.

Reply:
`802_1x` is also not a valid variable name in python, since the first character cannot be a number, so to access it the `[]` needs to be used anyhow, as in:

 `{{ network_connections[0]["802.1x"]["identity"] }}` or `{{ network_connections[0]["802_1x"]["identity"] }}`

If you still think it is a problem, please open an issue for this to track this and ideally provide a better name. Also if we want to have special restrictions here, it would make sense to add a test for this in the CI or the actual code to changes like this to get in (could be discussed in the ticket to be created).